src/Microsoft.Win32.Primitives/tests/Microsoft.Win32.Primitives.Tests.builds

```diff
@@ -8,6 +8,10 @@
     <Project Include="Microsoft.Win32.Primitives.Tests.csproj">
       <OSGroup>Windows_NT</OSGroup>
     </Project>
+    <Project Include="Microsoft.Win32.Primitives.Tests.csproj">
+      <TargetGroup>net46</TargetGroup>
```

Comment:
looks we need to think in more generic approach to split the concept of building from running. I don't think we support that today but well to think about.